service/src/test/java/uk/gov/hmcts/reform/fpl/utils/DocumentsHelperTest.java

```diff
@@ -49,6 +50,27 @@ void shouldReturnFalseIfDocumentHasNotBinaries() {
         assertThat(hasDocumentUploaded(document(false))).isEqualTo(false);
     }
 
+    @Test
+    void shouldConcatUrlAndDocumentPath() {
+        String invalidMostRecentDocumentLink = "http://dm-store:8080/documents/b28f859b-7521-4c84-9057-47e56afd773f/binary";
+        String url = "http://fake-url";
+
+        String documentUrl = concatUrlAndMostRecentUploadedDocumentPath(invalidMostRecentDocumentLink, url);
+
+        assertThat(documentUrl).isEqualTo("http://fake-url/documents/b28f859b-7521-4c84-9057-47e56afd773f/binary");
+    }
+
+
+    @Test
+    void shouldCatchExceptionAndReturnEmptyStringWhenDocumentLinkIsNotInCorrectFormat() {
+        String invalidMostRecentDocumentLink = "http://dm-store:8080/documents/b28f859b-7521-4c84-9057-47e56afd773f/binary^";
+        String url = "http://fake-url";
+
+        String documentUrl = concatUrlAndMostRecentUploadedDocumentPath(invalidMostRecentDocumentLink, url);
+
+        assertThat(documentUrl).isEqualTo("");
+    }
```

Comment:
I am actually not sure what's better here-  returning empty string or just. throwing the exception. It's helper class which shouldn't decide how the errors are handled, that should be calleer responsibility. But I guess I am happy in this instance. As it is only used there for email notification.

Reply:
Yes I can see what you mean here! I think in order to get this bug fix into testing asap let's agree to leave this for now (a couple of our other helpers actually throw exceptions / follow this same format) and if we wish to iterate back on it again we can. and yes only used for email notification so nothing major either!